framework/platform/bpl/uci/scripts/platform_daemon_ugw.sh

```diff
@@ -0,0 +1,146 @@
+    #2.4G
+    # 4addr-mode
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio26", "values" : {"wds" : "1"}}'
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio26", "values" : {"vendor_elems" : "13 dd050017353001 14 dd050017353001" } }'
+    # to remove above entry (use 3addr-mode) use: uci delete wireless.default_radio26.vendor_elems 
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio26", "values" : {"full_ch_master_control" : "1"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio26", "values" : {"disabled" : "0"}}'
+    
+    #5G
+    # 4addr-mode
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio58", "values" : {"wds" : "1"}}'
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio58", "values" : {"vendor_elems" : "13 dd050017353001 14 dd050017353001" } }'
+    
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio58", "values" : {"full_ch_master_control" : "1"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio58", "values" : {"disabled" : "0"}}'
+    
+    #uci commit wireless (currently automatically restarts the hostapd's)
+    ubus call uci commit '{"config" : "wireless" }'
+    # If the automatic restard is removed, the follwoing should be used after commit
+    #/etc/init.d/network reload
+}
+repeater_to_ap()
+{
+    # Kill APM
+    /opt/prplMesh/scripts/prplmesh_utils.sh stop
+    
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # Kill DHCP client
+	#kill `cat /var/run/udhcpc-br-lan.pid`
+
+    #configure APM
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"management_mode" : "Multi-AP-Controller-and-Agent"}}'
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"operating_mode" : "Gateway"}}'
+    ubus call uci commit '{"config" : "beerocks" }'
+    
+    #configure wireless
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio26", "values" : {"disabled" : "1"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio58", "values" : {"disabled" : "1"}}'
+    
+    #uci commit wireless (currently automatically restarts the hostapd's)
+    ubus call uci commit '{"config" : "wireless" }'
+    # If the automatic restard is removed, the follwoing should be used after commit
+    #/etc/init.d/network reload
+
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # Set bridge ip to 192.168.1.1
+    #echo "Set bridge ip to 192.168.1.1"
+    #ifconfig br-lan 192.168.1.1
+    
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # start DHCP server
+    #/etc/init.d/odhcpd start
+    
+    # Start APM
+    echo "Start APM"
+    /opt/prplMesh/scripts/prplmesh_utils.sh start
+}
+static_init()
```

Comment:
what is `static_init()`? should add a function description. Also please add a line break before starting a new function.